doc/filters.texi

```diff
@@ -13005,6 +13005,95 @@ lut2='if(lt(x,y),0,if(gt(x,y),pow(2,bdx)-1,pow(2,bdx-1))):if(lt(x,y),0,if(gt(x,y
 @end example
 @end itemize
 
+@section lvpdnn
+
+Inference the content in the input image/video by applying the lvpdnn methods based on
+convolutional neural networks. Supported models:
+
+TensorFlow model files(.pb)
+
+The filter accepts the following options:
+
+@table @option
+@item filter_type
+Specify which filter to use. This option accepts the following values:
+
+@table @samp
+@item lvpclassify
+lvpclassify filter. To conduct lvpclassify filter, you need to use a classify model.
+
+@item lvpodetect
+lvpodetect filter. To conduct object detect filter, you need to use a object detect model.
+@end table
+
+Default value is @samp{lvpclassify}.
+
+@item dnn_backend
+Specify which DNN backend to use for model loading and execution. This option accepts
+the following values:
+
+@table @samp
+@item native
+Native implementation of DNN loading and execution.
+
+@item tensorflow
+TensorFlow backend. To enable this backend you
+need to install the TensorFlow for C library (see
+@url{https://www.tensorflow.org/install/install_c}) and configure FFmpeg with
+@code{--enable-libtensorflow}
+@end table
+
+Default value is @samp{tensorflow}.
+
+@item model
+Set path to model file specifying network architecture and its parameters.
+Note that different backends use different file formats. TensorFlow can load files for only its format.
+
+@item input
+Set the input name of the dnn network.
+
+@item output
+Set the output name of the dnn network.
+
+@item sample
```

Comment:
Should we note that this should be a float in the 0.0-1.0 range?

Reply:
As a value corresponding to SampleRate, the filter is executed per sample frame. This is an integer bigger than 0.